.github/workflows/run-api-analytics-tests.yml

```diff
@@ -0,0 +1,79 @@
+name: Run analytics api tests
+env:
+  # This is to make sure Maven don't timeout fetching dependencies. See: https://github.com/actions/virtual-environments/issues/1499
+  MAVEN_OPTS: -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false -Dmaven.wagon.http.retryHandler.class=standard -Dmaven.wagon.http.retryHandler.count=3 -Dmaven.wagon.httpconnectionManager.ttlSeconds=125
+
+on:
+  pull_request:
+    types: [ opened, labeled, synchronize ]
+  schedule:
+    - cron: "16 1 * * *" # Run at 01:16 AM
+concurrency:
+  group: ${{ github.workflow}}-${{ github.ref }}
+  cancel-in-progress: true
+jobs:
+  api-test:
+    env:
+      CORE_IMAGE_NAME: "dhis2/core-dev:local"
+      PR_NUMBER: ${{ github.event.number }}
+      DOCKER_CHANNEL: "dhis2/core-pr"
+      SIERRA_LEONE_DB_PATH: "/tmp/db/sierra-leone"
+      SIERRA_LEONE_DB_VERSION: "2.39.0"
+    
+    runs-on: ubuntu-latest
+    if: "contains(github.event.pull_request.labels.*.name, 'run-api-analytics-tests')"
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          java-version: 11
+          distribution: temurin
+          cache: maven
+
+      - name: Build core image
+        run: |
+          mvn clean install --threads 2C --batch-mode --no-transfer-progress -DskipTests -Dmaven.test.skip=true -f dhis-2/pom.xml -pl -dhis-web-embedded-jetty,-dhis-test-integration,-dhis-test-coverage
+          mvn clean install --threads 2C --batch-mode --no-transfer-progress -DskipTests -Dmaven.test.skip=true -f dhis-2/dhis-web/pom.xml
+          mvn --batch-mode --no-transfer-progress -DskipTests -Dmaven.test.skip=true -f ./dhis-2/dhis-web/dhis-web-portal/pom.xml jib:dockerBuild -Djib.to.image=$CORE_IMAGE_NAME
+
+      # TODO why do we need the restore-keys?
```

Comment:
I do not think we actually need them in our case. As the key will be always the same "hardcoded" value.
I initially added them as they were part of the documentation. I believe we can safely remove them.